src/themes/teams/components/Grid/gridStyles.ts

```diff
@@ -14,7 +15,7 @@ const gridStyles: IComponentPartStylesInput = {
     props,
     variables: { height, width, defaultColumnCount, gridGap, padding },
   }: {
-    props: IProps
+    props: IGridProps
```

Comment:
I also think we should hold off on typing styles until we figure out how we're going to do that.  Probably warrants a separate PR, some testing, and discussion.